From: Mans Rullgard <mans@mansr.com>
Date: Fri, 29 Aug 2008 01:16:14 +0000 (+0100)
Subject: OMAP: Fix video timings info message
X-Git-Url: http://git.mansr.com/?p=linux-omap;a=commitdiff_plain;h=3a8bdf0967ae2c4eb3cebb97118ef0392f709c1c

OMAP: Fix video timings info message

This fixes the hsync frequency value printed on startup.

Signed-off-by: Mans Rullgard <mans@mansr.com>
---

diff --git a/drivers/video/omap/omapfb_main.c b/drivers/video/omap/omapfb_main.c
index d176a2c..e7f3462 100644
--- a/drivers/video/omap/omapfb_main.c
+++ b/drivers/video/omap/omapfb_main.c
@@ -1792,7 +1792,8 @@ static int omapfb_do_probe(struct platform_device *pdev,
 			vram, fbdev->mem_desc.region_cnt);
 	pr_info("omapfb: Pixclock %lu kHz hfreq %lu.%lu kHz "
 			"vfreq %lu.%lu Hz\n",
-			phz / 1000, hhz / 10000, hhz % 10, vhz / 10, vhz % 10);
+			phz / 1000, hhz / 10000, hhz % 10000,
+			vhz / 10, vhz % 10);
 
 	return 0;
 
